app/helpers/admin/orders_helper.rb

```diff
@@ -7,17 +7,23 @@ module OrdersHelper
     # We exclude shipping method adjustments because they are displayed in a
     # separate table together with the order line items.
     def order_adjustments_for_display(order)
-      adjustments_for_display = order.adjustments + order.all_adjustments.payment_fee.eligible
+      order.adjustments +
+        voucher_included_tax_representations(order) +
+        order.all_adjustments.payment_fee.eligible
```

Comment:
Should `voucher_included_tax_representations(order)` be the last one in the list? It looks like it was before.

Reply:
I thought that that was looking odd. It seems clearer to me to put the voucher tax under the voucher.